kitsune/inproduct/admin.py

```diff
@@ -1,13 +1,84 @@
+from django.conf import settings
 from django.contrib import admin
+from django.utils.html import format_html
+from django.utils.translation import gettext_lazy as _
 
 from kitsune.inproduct.models import Redirect
+from kitsune.wiki.models import Revision
+
+
+class KBLinkFilter(admin.SimpleListFilter):
+    title = _("KB article")
+    parameter_name = "kb_article"
+
+    def lookups(self, request, model_admin):
+        return (
+            ("with_kb", _("Has KB Article")),
+            ("without_kb", _("No KB Article")),
```

Comment:
Super Nit. I just realized that with my suggested changes, which always filter the `queryset` based for redirects targeted at KB articles, that perhaps these titles would be better as:
```suggestion
            ("with_kb", _("KB Article Exists")),
            ("without_kb", _("No KB Article Exists")),
```

Reply:
Sure thing.